XenModel/Messages.resx

```diff
@@ -8677,16 +8677,16 @@ Do you want to continue?</value>
     <value>Please eject your CD</value>
   </data>
   <data name="MIGRATION_NOT_ALLOWED" xml:space="preserve">
-    <value>This VM may not migrate at the moment</value>
+    <value>'{0}' may not migrate at the moment. Have you installed {1} on it?</value>

```

Comment:
Not sure I like the wording. Would something on the lines of `Please make sure you have installed {1} on the VM` work better?